iam_openapi_spec.yml

```diff
@@ -530,6 +713,36 @@ paths:
       security:
         - apiKeyAuth: []
 
+
+  /organizations/{organization_id}/sub_organizations/{sub_organization_id}/users/resetPassword:
```

Comment:
```suggestion
  /organizations/{organization_id}/sub_organizations/{sub_organization_id}/users/{user_name}/resetPassword:
```

Reply:
Resolving as this is not required. This API is triggered during forgot password flow.